### DISCUSSION TITLE
What is the different between Project and Edited?

### DISCUSSION THREAD

**Discussion by ghost** [fredvs/ideU#56] [Q&A] [answered]
What is the different between Project and Edited?
What does Make 1, Make 2,... and many other Make N means?

**Comment by fredvs** [comment, marked as answer]
> What is the different between Project and Edited?

The _Project_ regroup some units and has a main source file, for fpc the "program" or "library" unit.
That _Project_ has his own options.

The _Edited_ source is the active edited file in the editor.
Like _Geany IDE_ does, you may compile only this unit (not all the project).

In the _Commandor_ form, the _Project_ panel (light green) controls the project himself and the _Edited_ panel (light brown)  controls the active edited source.

![commander-ideu](https://user-images.githubusercontent.com/3421249/204336451-56588c24-2c67-477c-962b-3dfd43bc1625.png)


> What does Make 1, Make 2,... and many other Make N means?

In Project-options, you have _Make_, _Build_, _Make 0_ to _Make 9_ different options to compile the project.

![project-option-make-ideu](https://user-images.githubusercontent.com/3421249/204163388-242c6e69-b0f6-4186-a0f5-28f8e1e37534.png)

You may choose the make-option via menu _Project_ / _Make_.
Or, in the _Commandor_ form, use the pink list box in _Project_ light-green panel to select the make-option.

![commander-ideu](https://user-images.githubusercontent.com/3421249/204336451-56588c24-2c67-477c-962b-3dfd43bc1625.png)

The same if you want to compile only the active _Edited_ source.

**Comment by fredvs** [comment]
Since commit https://github.com/fredvs/ideU/commit/69b8e20bc110c77746218b7d511f6ff7954dd0c1 there is a option in Extra Settings to enable/disable this "Geany" way to compile only the edited source.
By default, this option is disable.
